tools/webpack.config.ts

```diff
@@ -131,6 +129,52 @@ export default (env?: string) : webpack.Configuration => {
         // re-evaluated as the size of files grows
         devtool: production ? 'source-map' : 'cheap-module-eval-source-map',
     };
+
+    // Run through and add expose-loader to the config
+    let exposeLoaders = [];
+    for(var exposeArr of exposeOptions) {
+        let path;
+        let name = "";
+        let nameArr;
+        let useArr = [];
+        if(typeof exposeArr === "string") {
+            path = exposeArr;
+            nameArr = exposeArr.split('/');
+            nameArr = nameArr[nameArr.length - 1];
+            nameArr = nameArr.split('.');
+            name = nameArr[nameArr.length - 2];
```

Comment:
This four lines of code if i interpreted correctly, is for getting `jquery` if the filename is `../../jquery.js`? If so this is what you should do.

```javascript
name = path.basename(path, '.js');
```

Couple of notes on it, you will need to import `path` module in full, i think currently it only imports resolve. And the variable `path` needs to be changed to `filePath` or `modulePath`.